datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -47,14 +48,21 @@ def manifest(ctx, check, fix):
 
             try:
                 decoded = json.loads(read_file(manifest_file).strip())
+                decoded = JSONDict(decoded)
             except json.JSONDecodeError as e:
                 failed_checks += 1
                 echo_info(f"{check_name}/manifest.json... ", nl=False)
                 echo_failure("FAILED")
                 echo_failure(f'  invalid json: {e}')
                 continue
 
+            version = decoded.get('manifest_version', '1.0.0')
+            all_validators = get_all_validators(ctx, version, is_extras, is_marketplace)
+
             for validator in all_validators:
+                if validator.skip_if_errors and file_failures > 0:
+                    echo_info(f'Skipping validation {validator} since errors have already been found.')
+                    continue
```

Comment:
👍 on continuing to list all validations that would be skipped instead of quickly aborting.